### PR TITLE
Update Indonesian translation

### PR DESCRIPTION
**Pull request by geomars**
Update Indonesian translation for GUI 0.13.1 or GUI 0.13.2

### CODE REVIEW

File: translations/monero-core_id.ts [hunk 1]
```diff
@@ -827,28 +827,28 @@
     <message>
         <location filename="../components/ProgressBar.qml" line="38"/>
         <source>%1 blocks remaining: </source>
-        <translation type="unfinished"></translation>
+        <translation>% 1 blok tersisa:</translation>
```
Comment (erciccione) [inline translations/monero-core_id.ts]: Please remove the space between `%` and `1`

File: translations/monero-core_id.ts [hunk 2]
```diff
@@ -2504,353 +2501,346 @@ For the case with Spend Proof, you don&apos;t need to specify the recipient addr
...
-        <location filename="../main.qml" line="430"/>
+        <location filename="../main.qml" line="429"/>
         <source>Unlocked balance</source>
         <translation>Saldo rekening yang tidak terkunci</translation>
     </message>
     <message>
-        <location filename="../main.qml" line="515"/>
+        <location filename="../main.qml" line="514"/>
         <source>Waiting for daemon to start...</source>
         <translation>Menunggu jurik untuk memulai</translation>
```
Comment (erciccione) [inline translations/monero-core_id.ts]: please add `...`

File: translations/monero-core_id.ts [hunk 3]
```diff
@@ -2504,353 +2501,346 @@ For the case with Spend Proof, you don&apos;t need to specify the recipient addr
...
     </message>
     <message>
-        <location filename="../main.qml" line="521"/>
+        <location filename="../main.qml" line="520"/>
         <source>Waiting for daemon to stop...</source>
         <translation>Menunggu jurik untuk berhenti</translation>
```
Comment (erciccione) [inline translations/monero-core_id.ts]: Please add `...`

File: translations/monero-core_id.ts [hunk 4]
```diff
@@ -2504,353 +2501,346 @@ For the case with Spend Proof, you don&apos;t need to specify the recipient addr
...
         <translation>Jumlah: </translation>
     </message>
     <message>
-        <location filename="../main.qml" line="636"/>
-        <location filename="../main.qml" line="754"/>
+        <location filename="../main.qml" line="635"/>
+        <location filename="../main.qml" line="753"/>
         <source>
 Fee: </source>
         <translation>Biaya</translation>
```
Comment (erciccione) [inline translations/monero-core_id.ts]: Please add `:` + whitespace

File: translations/monero-core_id.ts [hunk 5]
```diff
@@ -2504,353 +2501,346 @@ For the case with Spend Proof, you don&apos;t need to specify the recipient addr
...
         <source>Error: Filesystem is read only</source>
-        <translation type="unfinished"></translation>
+        <translation>Kesalahan: Filesystem hanya bisa dibaca</translation>
     </message>
     <message>
-        <location filename="../main.qml" line="1151"/>
+        <location filename="../main.qml" line="1150"/>
         <source>Warning: There&apos;s only %1 GB available on the device. Blockchain requires ~%2 GB of data.</source>
-        <translation type="unfinished"></translation>
+        <translation>Catatan: Hanya %1 GB yang tersedia di perangkat. Blockchain membutuhkan data ~% 2 GB.</translation>
```
Comment (erciccione) [inline translations/monero-core_id.ts]: remove space between `%` and `2`

File: translations/monero-core_id.ts [hunk 6]
```diff
@@ -2504,353 +2501,346 @@ For the case with Spend Proof, you don&apos;t need to specify the recipient addr
...
     </message>
     <message>
-        <location filename="../main.qml" line="1153"/>
+        <location filename="../main.qml" line="1152"/>
         <source>Note: There&apos;s %1 GB available on the device. Blockchain requires ~%2 GB of data.</source>
-        <translation type="unfinished"></translation>
+        <translation>Catatan: Hanya %1 GB yang tersedia di perangkat. Blockchain membutuhkan data ~% 2 GB.</translation>
```
Comment (erciccione) [inline translations/monero-core_id.ts]: remove space between `%` and `2`

File: translations/monero-core_id.ts (hunk 3 above)
Comment (geomars) [reply translations/monero-core_id.ts]: ok

**Comment by erciccione** [comment]
Why did you close this PR?

**Comment by geomars** [comment]
Oh it's accidental not intentional, I will reopen it, just because I've done reset hard first then do `lupdate -pro monero-wallet-gui.pro -ts translations/monero-core_id.ts -no-obsolete`

**Comment by erciccione** [comment]
Looks like the commit is gone. Could you recommit and reopen the PR?

**Comment by geomars** [comment]
Already did with PR #1687

**Pull request closed without merging**